raiden/tests/integration/test_matrix_transport.py

```diff
@@ -573,3 +573,475 @@ def test_matrix_send_global(
 
     transport.stop()
     transport.get()
+
+
+def test_matrix_invite_private_room_happy_case(
```

Comment:
As we talked about in the call this will not actually run with private rooms. 
Either override the `private_rooms` fixture with `parametrize` or use the `public_and_private_rooms` fixture to test both cases.

Reply:
Done.

Reply:
Hm it would be cleaner to extend the `private_rooms` fixture so it knows about the number of servers / transports and then parametrize that.